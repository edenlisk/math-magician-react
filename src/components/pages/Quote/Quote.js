import React from 'react';

const Quote = () => (
  <div className="container-fluid text-center py-5">
    <p className="fw-bold h5">
      Mathematics is not about numbers, equations, computations,
      <br />
      or algorithms: it is about understanding. -William Paul Thurston
    </p>
  </div>
);

export default Quote;
