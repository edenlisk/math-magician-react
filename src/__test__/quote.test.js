import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import TestRenderer from 'react-test-renderer';
import Quote from '../components/pages/Quote/Quote';

describe('Test Quote components', () => {
  test('It should render Quote component', () => {
    render(<Quote />);
    expect(screen.getByText(/William Paul Thurston/)).toBeInTheDocument();
  });

  test('It should render correctly', () => {
    const tree = TestRenderer.create(<Quote />);
    expect(tree).toMatchSnapshot();
  });
});
