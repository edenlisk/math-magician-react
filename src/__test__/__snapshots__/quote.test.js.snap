// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Quote components It should render correctly 1`] = `
<div
  className="container-fluid text-center py-5"
>
  <p
    className="fw-bold h5"
  >
    Mathematics is not about numbers, equations, computations,
    <br />
    or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
